eo-phi-normalizer/src/Language/EO/Phi/Syntax/Doc.txt

```diff
@@ -1,132 +1,132 @@
-
-BytesMetaId literals are recognized by the regular expression
-`````{"!y"} (char - ["
-
 !'(),-.:;?[]{|}⟦⟧"])*`````
-
-MetaFunctionName literals are recognized by the regular expression
-`````'@' (char - ["
-
 !'(),-.:;?[]{|}⟦⟧"])*`````
-
-
-===Reserved words and symbols===
-The set of reserved words is the set of terminals appearing in the grammar. Those reserved words that consist of non-letter characters are called symbols, and they are treated in a different way from those that are similar to identifiers. The lexer follows rules familiar from languages like Haskell, C, and Java, including longest match and spacing conventions.
-
-The reserved words used in Syntax are the following:
-  | ``Δ`` | ``Φ`` | ``λ`` | ``ξ``
-  | ``ρ`` | ``φ`` |  |
-
-The symbols used in Syntax are the following:
-  | { | ⟦ | ⟧ | }
-  | ( | ) | . | ⊥
-  | [ | ↦ | ] | *
-  | ∅ | ⤍ | , |
-
-===Comments===
-Single-line comments begin with //.Multiple-line comments are  enclosed with /* and */.
-
-==The syntactic structure of Syntax==
-Non-terminals are enclosed between < and >.
-The symbols -> (production),  **|**  (union)
-and **eps** (empty rule) belong to the BNF notation.
-All other symbols are terminals.
-
-  | //Program// | -> | ``{`` ``⟦`` //[Binding]// ``⟧`` ``}``
-  | //MetaId// | -> | //LabelMetaId//
-  |  |  **|**  | //TailMetaId//
-  |  |  **|**  | //BindingsMetaId//
-  |  |  **|**  | //ObjectMetaId//
-  |  |  **|**  | //BytesMetaId//
-  | //Object// | -> | ``⟦`` //[Binding]// ``⟧``
-  |  |  **|**  | //Object// ``(`` //[Binding]// ``)``
-  |  |  **|**  | //Object// ``.`` //Attribute//
-  |  |  **|**  | ``Φ``
-  |  |  **|**  | ``ξ``
-  |  |  **|**  | ``⊥``
-  |  |  **|**  | //Object// ``[`` ``ξ`` ``↦`` //Object// ``]``
-  |  |  **|**  | //ObjectMetaId//
-  |  |  **|**  | //Object// ``*`` //TailMetaId//
-  |  |  **|**  | //MetaFunctionName// ``(`` //Object// ``)``
-  | //Binding// | -> | //Attribute// ``↦`` //Object//
-  |  |  **|**  | //Attribute// ``↦`` ``∅``
-  |  |  **|**  | ``Δ`` ``⤍`` //Bytes//
-  |  |  **|**  | ``Δ`` ``⤍`` ``∅``
-  |  |  **|**  | ``λ`` ``⤍`` //Function//
-  |  |  **|**  | //BindingsMetaId//
-  |  |  **|**  | ``Δ`` ``⤍`` //BytesMetaId//
-  | //[Binding]// | -> | **eps**
-  |  |  **|**  | //Binding//
-  |  |  **|**  | //Binding// ``,`` //[Binding]//
-  | //Attribute// | -> | ``φ``
-  |  |  **|**  | ``ρ``
-  |  |  **|**  | //LabelId//
-  |  |  **|**  | //AlphaIndex//
-  |  |  **|**  | //LabelMetaId//
-  | //RuleAttribute// | -> | //Attribute//
-  |  |  **|**  | ``Δ``
-  |  |  **|**  | ``λ``
-  | //PeeledObject// | -> | //ObjectHead// //[ObjectAction]//
-  | //ObjectHead// | -> | ``⟦`` //[Binding]// ``⟧``
-  |  |  **|**  | ``Φ``
-  |  |  **|**  | ``ξ``
-  |  |  **|**  | ``⊥``
-  | //ObjectAction// | -> | ``(`` //[Binding]// ``)``
-  |  |  **|**  | ``.`` //Attribute//
-  | //[ObjectAction]// | -> | **eps**
-  |  |  **|**  | //ObjectAction// //[ObjectAction]//
-
-
-
-%% File generated by the BNF Converter (bnfc 2.9.5).
+The Language Syntax
```

Comment:
Why is this file changed?

Reply:
Did you change file encoding?

Reply:
This change applied automatically when I pulled and built master